bitnami/rabbitmq/values.yaml

```diff
@@ -511,8 +514,10 @@ configuration: |-
   {{- else if (eq .Values.memoryHighWatermark.type "relative") }}
   vm_memory_high_watermark.{{ .Values.memoryHighWatermark.type }} = {{ .Values.memoryHighWatermark.value }}
   {{- end }}
+  {{- end }}
   ## TCP Listen Options
   ##
+  {{- if .Values.tcpListenOptions.enabled }}
```

Comment:
This "if" should be included before the comment in line 518

Reply:
Updated